src/app/components/register/register.component.scss

```diff
@@ -0,0 +1,58 @@
+form {
+  padding-top: 20px;
+}
+
+.row {
```

Comment:
in Bootstrap "row" has a built-in -5 (or 10) px margin only on the right side so there was a white line on the right of the page on desktop (and on mobile view, too). This was the only option to get rid of that white line. Without the "row" class, it would have been more complex to create the style of the page